src/acl/ConnMark.cc

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (C) 1996-2017 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+/* DEBUG: section 28    Access Control */
+
+#include "squid.h"
+#include "acl/FilledChecklist.h"
+#include "acl/ConnMark.h"
+#include "cache_cf.h"
+#include "Debug.h"
+
```

Comment:
Please remove this empty line and move the Debug.h include below the c* filenames. These lists should be alphabetical.